synapse/lib/storm.py

```diff
@@ -225,33 +225,74 @@ def __len__(self):
 
     def size(self):
         '''
-        Get the number of tufos currently in the query.
+        Get the number of nodes currently in the query.
+
+        Returns:
+            int: Number of nodes currently in the query.
         '''
         return len(self)
 
     def log(self, **info):
         '''
-        Log execution metadata for the current oper.
+        Log execution metadata for the current user.
```

Comment:
this is def for the "current oper"  ( the oplog[-1] thing is being used to show *which* oper caused a log entry )